IQSS/dataverse
# 6748 configure dropdown int

**What this PR does / why we need it**: Internationalization of DCT

**Which issue(s) this PR closes**: #6748 

**How to test**: 
In foreign bundles, for example in Bundle_fr.properties add

`externaltools.dct.displayname=Data Curation Tool French`
`externaltools.dct.description=Data Curation Tool for curation of variables French`
`externaltools.explorer.displayname=Data Explorer French`

Deploy new code.

Into manifest json of DCT [https://github.com/scholarsportal/Dataverse-Data-Curation-Tool/blob/master/DataCuration_v4.17up.json](https://github.com/scholarsportal/Dataverse-Data-Curation-Tool/blob/master/DataCuration_v4.17up.json) add field `toolName: "dct"`
Into manifest json of DataExplorer  https://github.com/scholarsportal/Dataverse-Data-Explorer/blob/master/dataExplorer_v4.17up.json add field  `toolName: "explorer"`.
Run `curl -X POST -H 'Content-type: application/json' --upload-file manifest.json http://localhost:8080/api/admin/externalTools` for corresponding DataCuration_v4.17up.json and dataExplorer_v4.17up.json instead of manifest.json

Go to ingested tab file, switching the langage should change the "Configure" and "Explore" dropdown box. For "Explore" one will need to have at least two explorer tools to have a dropdown box. For "Configure" were will be a popup window with corresponding description after selecting the tool. There is no popup for explorer.
